crs_motion_planning/src/freespace_planning_server.cpp

```diff
@@ -222,7 +230,7 @@ class PlanningServer: public rclcpp::Node
     traj_pc->smooth_velocities = true;
     traj_pc->smooth_accelerations = true;
     traj_pc->smooth_jerks = true;
-    traj_pc->longest_valid_segment_length = 0.1;
+    traj_pc->longest_valid_segment_length = 0.05;
```

Comment:
Should this be made into a configurable parameter?

Reply:
In the next iteration of the full process planner I plan to make it more configurable, but I think this freespace planner node is okay how it is now.